lib/components/button-group/button-group.visual.test.ts

```diff
@@ -0,0 +1,137 @@
+import { html } from "@open-wc/testing";
+import { runVisualTests } from "../../test/visual-test-utils";
+import "../../index";
+
+const btns = [
+    { name: "Newest", isSelected: true },
+    { name: "Frequent" },
+    { name: "Active" },
+    { name: "Bountied" },
+    { name: "Unanswered" },
+];
+
+const form = (children: number[]) => `
```

Comment:
We can remove those variants since they don't add much value, screenshots are identical